sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 ### 4.3.1 (Unreleased)
 
 #### Features Added
-
+- Added the ability to create containers and databases with autoscale properties for the sync and async clients. - [See PR #24976](https://github.com/Azure/azure-sdk-for-python/pull/24976)
```

Comment:
@annatisch do we typically link to PRs in the change log? I've seen where we might link to a particular GH issue when fixing something.

Reply:
I got the idea from Simon so I am not too sure myself.

Reply:
Per our discussion, let's not extend this pattern to Python. If you are linking to a Github issue for a bug fix, that is fine (nd encouraged).